### PR TITLE
tests: add test case for mode value none

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Tests.

improved mode tests and added a missing test case for `--mode none`.

**Did you add tests for your changes?**

yes

**If relevant, did you update the documentation?**
NA

**Summary**

Added missing tests case for `--mode none`.

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

File: test/mode/mode-single-arg/mode-single-arg.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        expect(stdout).toBeTruthy();
+
+        stat(resolve(__dirname, './bin/main.js'), (err, stats) => {
+            expect(err).toBe(null);
+            expect(stats.isFile()).toBe(true);
+            done();
+        });
+    });
+
+    it('should load a production config when --mode=productuon is passed', (done) => {
```
Comment (jamesgeorge007) [inline test/mode/mode-single-arg/mode-single-arg.test.js:18]: Typo.

Comment (snitin315) [reply test/mode/mode-single-arg/mode-single-arg.test.js:18]:
```suggestion
    it('should load a production config when --mode=production is passed', (done) => {
```

**Comment by evilebottnawi** [comment]
Thanks!
